app/policies/user_policy.rb

```diff
@@ -8,4 +8,8 @@ def permitted_attributes_for_create
   def permitted_attributes_for_update
     [:email]
   end
+
+  def drafts?
```

Comment:
looks good! I wonder if it's worth making this method name a bit more descriptive and reflect the code inside it?